activiti-core/activiti-engine/src/main/java/org/activiti/engine/impl/bpmn/behavior/SequentialMultiInstanceBehavior.java

```diff
@@ -81,13 +81,16 @@ public void leave(DelegateExecution childExecution) {
     setLoopVariable(multiInstanceRootExecution, NUMBER_OF_COMPLETED_INSTANCES, nrOfCompletedInstances);
     setLoopVariable(childExecution, getCollectionElementIndexVariable(), loopCounter);
     logLoopDetails(childExecution, "instance completed", loopCounter, nrOfCompletedInstances, nrOfActiveInstances, nrOfInstances);
+
+    updateResultCollection(childExecution, multiInstanceRootExecution);
     
     Context.getCommandContext().getHistoryManager().recordActivityEnd((ExecutionEntity) childExecution, null);
     callActivityEndListeners(childExecution);
     
     //executeCompensationBoundaryEvents(execution.getCurrentFlowElement(), execution);
```

Comment:
is still needed this commented code?

Reply:
Commented code is never needed :). I removed it.